pages/common/webpack.md

```diff
@@ -0,0 +1,19 @@
+# webpack
+
+> Bundle your js files and other assets into a single output file.
+
+- Create a single output file from an entry point file:
+
+`webpack {{app.js}} {{bundle.js}}`
+
+- Load css files too from your js file. (This uses the css loader for .css files):
+
+`webpack {{app.js}} {{bundle.js}} --module-bind 'css=css'`
+
+- Pass a config file and show compilation progress:
+
+`webpack --config {{webpack.config.js}} --progress`
```

Comment:
what's the bare minimum such a file must have to work? I would say just list the properties included in the file, e.g. the entry point and the output filename.